x-pack/dockerlogbeat/readme.md

```diff
@@ -12,6 +12,22 @@ To build and install, just run `mage Package`. The build process happens entire
 `docker run --log-driver=elastic-logging-plugin:8.0.0 --log-opt output.elasticsearch.hosts="172.18.0.2:9200" --log-opt output.elasticsearch.index="dockerbeat-test" -it debian:jessie /bin/bash`
 
 
+## Config Options
+
+| Option               | Description                                                                                                                       |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
I'm not sure whether the formatting of this table is quite right because I don't know markdown, but it seems like you also need some kind of end delimiter.